tldw_Server_API/app/api/v1/endpoints/watchlists.py

```
@@ -943,7 +945,8 @@ async def bulk_create_sources(
                     )
                     errors_count += 1
                     continue
-        except Exception:
+        except Exception as exc:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)